tree_hash/Cargo.toml

```diff
@@ -13,13 +13,14 @@ categories = ["cryptography::cryptocurrencies"]
 [dependencies]
 ethereum-types = "0.14.1"
 ethereum_hashing = "0.6.0"
+ethereum_ssz = { git = "https://github.com/macladson/ethereum_ssz", branch = "bitvector"  }
```

Comment:
Once `ethereum_ssz` is merged, this should be switched to whichever release contains the changes. Same for `ethereum_ssz_derive` below.